### PR TITLE
Add documentation link to NuGet package descriptions

### PR DESCRIPTION
**Pull request by jcouv**
Doc-only change to 3 compiler nuget packages.

**Customer scenario**

Navigate to https://www.nuget.org/packages/Microsoft.Net.Compilers/ or another compiler package. There you will see a 1-line description of the package.
This PR adds a link to our wiki, so that we can document packages more thoroughly.
For instance, see this [question](https://stackoverflow.com/questions/42583579/what-version-of-the-compiler-is-in-a-microsoft-net-compilers-package) about the correspondance between package version and language version.

:memo: The nuspec format only allows plain text in the `<description>`. 

@dotnet/roslyn-compiler @jasonmalinowski for review.

Relates to https://github.com/dotnet/roslyn/issues/19659

### CODE REVIEW

File: src/NuGet/Microsoft.CodeAnalysis.CSharp.nuspec [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <id>Microsoft.CodeAnalysis.CSharp</id>
     <description>
       .NET Compiler Platform ("Roslyn") support for C#, Microsoft.CodeAnalysis.CSharp.dll.
+
+      More details at https://github.com/dotnet/roslyn/wiki/NuGet-packages
```
Comment (jasonmalinowski) [inline src/NuGet/Microsoft.CodeAnalysis.CSharp.nuspec]: Let's use an aka.ms link here so we can redirect for packages we've already shipped.

Comment (jcouv) [inline src/NuGet/Microsoft.CodeAnalysis.CSharp.nuspec]: Good idea. Thanks
Do we have a list or a known set of owners for aka.ms managed by Roslyn team?

**Comment by jcouv** [comment]
@jasonmalinowski Good to go?

**Comment by jcouv** [comment]
@agocke Could you review as well? Thanks

**Comment by jcouv** [comment]
@MeiChin-Tsai This is adding a link to the documentation of our nuget packages.
I don't think this requires ask-mode approval, since no code change, but I want to confirm with you. Thanks

**Pull request merged**